bin/i18n/hourofcode_crowdin.yml

```diff
@@ -15,28 +15,19 @@
 #
 files: [
   {
-    "source": "/en.yml",
-    "translation": "/%language%/hourofcode/%original_file_name%",
-    "update_option": "update_as_unapproved"
-  }, {
-    "source": "/how-to/*.md",
+    "source": "/how-to/*",
```

Comment:
it seems kind of unnecessary to include each subdirectory individually, and I don't love including everything and excluding only `.md.erb` files; in particular, this would sync files that are just `.erb`.

What if instead we do something like

```yaml
#
# Files configuration
#
files: [
  {
    "source": "/en.yml",
    "translation": "/%language%/hourofcode/%original_file_name%",
    "update_option": "update_as_unapproved"
  }, {
    "source": "/**/*.md",
    "translation": "/%language%/hourofcode/**/%original_file_name%",
    "update_option": "update_as_unapproved"
  }, {
    "source": "/**/*.md.partial",
    "translation": "/%language%/hourofcode/**/%original_file_name%",
    "update_option": "update_as_unapproved"
  }
]
```